### DISCUSSION TITLE
My chart doesn't display correctly

### DISCUSSION THREAD

**Discussion by Steellgold** [shadcn-ui/ui#5532] [Q&A] [answered]
Hello,

I'm facing an issue where my chart does not display, and I receive errors in the console when trying to render it. The error I'm seeing is:

```console
Failed to get source map: [Error: Unknown url scheme] { code: 'GenericFailure' }
```

I’ve even rebuilt a project from scratch, but nothing seems to work. I’ve tried running the project on both my local PC and a GitHub Codespace, but the results are the same — the chart doesn’t display. However, the surrounding card renders fine.

Here’s an example of the code I’m using:

```jsx
<CardFooter>
  <ChartContainer config={chartConfig} className="min-h-[200px] w-full">
    <BarChart accessibilityLayer data={chartData}>
      <Bar dataKey="desktop" fill="var(--color-desktop)" radius={4} />
      <Bar dataKey="mobile" fill="var(--color-mobile)" radius={4} />
    </BarChart>
  </ChartContainer>
</CardFooter>
```

I've double-checked my chartConfig and chartData, and everything seems to be correct, but the chart still doesn't show up.

Do you have any ideas or suggestions on how to resolve this issue?

I'm on NextJS 15, and as written in the documentation, I use recharts@alpha

![image](https://github.com/user-attachments/assets/9b307e65-ad96-4efe-a21f-08d6ffe256db)
![image](https://github.com/user-attachments/assets/aebd0fc5-cf86-4f32-a2d4-f2c7a653e68c)

Thank you in advance for your help!

**Comment by tejjyassin** [comment]
Same issue, Have you found a solution?

**Reply by Steellgold** [reply]
No

**Reply by 0xEthanS** [reply]
I am also having issues with the charts. Are you guys using Next15 or React19? I'm working through it, but React19 is only in canary release, and comes as the default version with Next15, and breaks a lot of packages like Radix-Icons, Recharts that Shadcn relies on, and React-is that recharts relies on.

**Comment by onefaceID** [comment]
I have a samedi problème

**Comment by Steellgold** [comment, marked as answer]
https://ui.shadcn.com/docs/react-19#recharts
